workflow/artifacts/s3/s3.go

```diff
@@ -72,7 +75,7 @@ func (s3Driver *S3ArtifactDriver) Load(inputArtifact *wfv1.Artifact, path string
 func (s3Driver *S3ArtifactDriver) Save(path string, outputArtifact *wfv1.Artifact) error {
```

Comment:
As discussed on Slack, would it be worth adding an info log to stat the file size for debug purposes, in the event that we see a discrepancy between the file on disk and the file in the S3 bucket?